.github/working-dir/simple.py

```diff
@@ -1,5 +1,7 @@
 import daft
 
+daft.context.set_runner_ray("ray://localhost:10001")
```

Comment:
This shouldn't be needed if DAFT_RUNNER=Ray and vice versa

Rather than throwing everything at the wall and seeing what sticks, let's try to see if there's a bug here that we should fix?

Reply:
Makes sense. I'll remove this context-setter from `simple.py` and only pass in `DAFT_RUNNER=ray` through the shell script in the GHA step.

Reply:
Yeah and it *still* not picking up that it's running inside a Ray job is definitely a bug that we should fix

Reply:
Yup, confirmed that after removing that `daft.context.set_runner_ray(...)`, the daft logs are still not being produced.

Source:
https://github.com/Eventual-Inc/Daft/actions/runs/11989916689